src/Console/SshCommand.php

```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```

Comment:
atm I can't think of a good guess. using the only one defined sounds straight forward.

in  a interactive mode you might "ask the user" to choose from the defined list, but non-interactive I dont think you can pre-dict which host is the "correct" one.